exe/wallet/Main.hs

```diff
@@ -104,7 +111,8 @@ Usage:
   cardano-wallet list wallets [--port=INT]
   cardano-wallet get wallet --wallet-id=STRING [--port=INT]
   cardano-wallet create wallet --name=STRING [--address-pool-gap=INT] [--port=INT]
-  cardano-wallet delete wallet --id=STRING
+  cardano-wallet delete wallet --wallet-id=STRING [--port=INT]
+  cardano-wallet create transaction --wallet-id=STRING --target=STRING... [--port=INT]
```

Comment:
Could we call it "submit transaction" instead of create?

Reply:
Hmmm.. The CLI is dual with the API, and here, we follow the semantic for HTTP verbs. So, `create` corresponds to `POST`.

Reply:
it is "transaction create" atm - I am ok with "transaction submit" as well if we decide to. Atm it corresponds to POST

Reply:
I don't think we need to use CRUD jargon in the CLI just because it makes HTTP calls to the backend.

Reply:
Why would "submit" be better than "create" here? Is the transaction something we already have in our hands, that we simply need to submit to the backend? Or does it actually cause the creation of a transaction according the parameters we supply?

Reply:
Well, it's actually a three step process:

1/ Create the transaction by performing a coin selection
2/ Sign the transaction
3/ Submit the transaction 

Perhaps there's a good name that captures all this, especially since we're planning to expose those three steps separately to offer the choice of doing some of them off-wallet.

Reply:
make, execute, css (create sign submit)

we might also opt to do all 3 steps above if no verb has been supplied.
```
transaction create => creates transaction
transaction sign => signs the transaction
transaction submit => submits the transaction to the network
transaction => all steps above
```

Reply:
moved to blackboard